src/e2e/java/uk.gov.hmcts.reform.sscs/olde2e/ProcessFileAndSaveIntoCcd.java

```diff
@@ -1,4 +1,4 @@
-package uk.gov.hmcts.reform.sscs.functional;
+package uk.gov.hmcts.reform.sscs.olde2e;
```

Comment:
why is there an `olde2e` package - does this imply the tests will be deleted? If so, shall we just delete them?

Reply:
This is there because, once we move all the tests using new framework we will delete them later